redbot/cogs/mod/mod.py

```diff
@@ -104,20 +106,24 @@ async def _maybe_update_config(self):
                 await self.config.guild(discord.Object(id=guild_id)).delete_repeats.set(val)
             await self.config.version.set("1.0.0")  # set version of last update
         if await self.config.version() < "1.1.0":
-            msg = _(
-                "Ignored guilds and channels have been moved. "
-                "Please use `[p]moveignoredchannels` if "
-                "you were previously using these functions."
-            )
-            self.bot.loop.create_task(send_to_owners_with_prefix_replaced(self.bot, msg))
+            async for e in AsyncIter((await self.config.all_channels()).values(), steps=25):
+                if e["ignored"] is not False:
+                    msg = _(
+                        "Ignored guilds and channels have been moved. "
+                        "Please use `[p]moveignoredchannels` to migrate the old settings."
+                    )
+                    self.bot.loop.create_task(send_to_owners_with_prefix_replaced(self.bot, msg))
+                    break
```

Comment:
Both guilds and channels can be ignored - we need code that checks guilds as well.